WowUp.WPF/Extensions/AddonExtensions.cs

```diff
@@ -108,5 +109,7 @@ public static bool Matches(this Addon addon1, Addon addon2)
                 addon1.ProviderName == addon2.ProviderName &&
                 addon1.ClientType == addon2.ClientType;
         }
+
+        public static string GetFullFolderPath(this Addon addon, IWarcraftService warcraftService) => Path.GetFullPath(Path.Combine(warcraftService.GetAddonFolderPath(addon.ClientType), addon.FolderName));
```

Comment:
This should probably go in the IWarcraftService
_Originally posted by @jliddev in https://github.com/jliddev/WowUp/pull/64#discussion_r496839280_

---

Do you want an addon feature to be in there?

Reply:
Good point, AddonService is probably more appropriate since it already references Warcraft service. Having it be an extension seems out of place with dependency injection.